src/main/scala/nl/knaw/dans/bag/v0/DansV0Bag.scala

```diff
@@ -477,13 +477,20 @@ class DansV0Bag private(private[v0] val locBag: LocBag) extends DansBag {
   override def addPayloadFile(src: File, pathInData: Path)
                              (implicit importOption: ImportOption): Try[DansV0Bag] = Try {
     val dest = data / pathInData.toString
-    mustNotExistInBag(dest)
-    mustBeChildOfBagData(dest)
+    if (dest == data && importOption == COPY) {
+      // allow to copy a directory to an empty data directory
+      if (!src.isDirectory || dest.list.nonEmpty)
```

Comment:
can't you do something like `dest.isEmpty`? Not sure... Maybe look in the source code or documentation of better-files.